test/features/pagecontent/save_api.js

```diff
@@ -11,7 +11,9 @@ describe('page save api', function() {
     var uri = server.config.bucketURL + '/wikitext/User:Mobrovac-WMF%2FRB_Save_Api_Test';
     var htmlUri = server.config.bucketURL + '/html/User:Mobrovac-WMF%2FRB_Save_Api_Test';
     var token = '';
-    var saveText = "Welcome to the page which tests the RESTBase save API!\n\n" +
+    var saveText = "Welcome to the page which tests the [[:mw:RESTBase|RESTBase]] save" +
+        "API! This page is created by an automated test to make sure RESTBase works " +
```

Comment:
You might want to insert a space before " API.., otherwise it'll read as 'saveAPI'.